packages/nextjs-mf/src/loaders/delegateLoader.ts

```diff
@@ -0,0 +1,44 @@
+import type {LoaderContext} from 'webpack';
+import path from 'path';
+
+/**
+ *
+ * Requires `include-defaults.js` with required shared libs
+ *
+ */
+export default function patchDefaultSharedLoader(
+  this: LoaderContext<Record<string, unknown>>,
+  content: string
+) {
+
+  const {delegates} = this.getOptions() as Record<string, string>;
+
+  const resolvedDelegates = Object.values(delegates).map((delegate) => {
+    const [request, query] = delegate.replace('internal ','').split('?')
+    return path.resolve(this._compiler?.context || '',request) + '?remote=' + new URLSearchParams(query).get('remote')
+  })
+
+
+  const requiredDelegates = resolvedDelegates.map((delegate) => {
```

Comment:
write delegates into files somewhere, since webpack needs internal refs (internal ./path) to be in the graph before its called. So we need to require() the delegate to make webpack bundle it into the graph.